spread.yaml

```diff
@@ -8,25 +8,30 @@ environment:
 include:
     - tests/
     - tools/
+    - docs/
+    - requirements-doc.txt
+    - requirements-focal.txt
+    - Makefile
+    - rockcraft/
 
 backends:
   google:
     key: '$(HOST: echo "$SPREAD_GOOGLE_KEY")'
     location: snapd-spread/us-east1-b
     halt-timeout: 2h
     systems:
-      - ubuntu-18.04-64:
-          workers: 1
-          storage: 40G
-      - ubuntu-20.04-64:
-          workers: 1
-          storage: 40G
+      # - ubuntu-18.04-64:
```

Comment:
are these temporarily disabled?

Reply:
same as above...forgot to undo my local test configurations. fixing it